docs/using.md

```diff
@@ -29,20 +29,20 @@ This section contains information about basic virtual machine and OpenShift mana
 <a name="starting-openshift"></a>
 ### Starting OpenShift
 
-The [minishift start](./docs/minishift_start.md) command can be used to start your OpenShift instance.
+The [minishift start](minishift_start.md) command is used to start your OpenShift instance.
 This command creates and configures a virtual machine that runs a single-node OpenShift instance.
 
 <a name="stopping-openshift"></a>
 ### Stopping OpenShift
 
-The [minishift stop](./docs/minishift_stop.md) command can be used to stop your OpenShift instance.
-This command shuts down the Minishift virtual machine, but preserves all instance state and data.
+The [minishift stop](minishift_stop.md) command is used to stop your OpenShift instance.
+This command shuts down the Minishift virtual machine, but preserves the instance state and data.
```

Comment:
hmm, is it not 'instance's' in this case? Genitive!? Personally I feel 'all' read better.